frames/src/main/java/com/checkout/frames/di/module/StylesModule.kt

```diff
@@ -89,5 +95,18 @@ internal abstract class StylesModule {
             containerMapper: Mapper<ContainerStyle, Modifier>
         ): Mapper<InputComponentStyle, InputComponentViewStyle> =
             InputComponentStyleToViewStyleMapper(textLabelStyleMapper, inputFieldStyleMapper, containerMapper)
+
+        @Provides
+        fun provideCardSchemeComponentStyleMapper(
+            textLabelStyleMapper: Mapper<TextLabelStyle, TextLabelViewStyle>,
+            containerMapper: Mapper<ContainerStyle, Modifier>
+        ): Mapper<CardSchemeComponentStyle, CardSchemeComponentViewStyle> =
+            CardSchemeComponentStyleToViewStyleMapper(textLabelStyleMapper, containerMapper)
+
+        @Provides
+        fun provideCardSchemeComponentStateMapper(
+            textLabelMapper: Mapper<TextLabelStyle?, TextLabelState>
+        ): Mapper<CardSchemeComponentStyle, CardSchemeComponentState> =
+            CardSchemeComponentStyleToStateMapper(textLabelMapper)
```

Comment:
Those mappers can't be reused in any other component, so it would be better to create a module for the subcomponent and provide them there.
Please take a look at the [Dagger2 doc](https://dagger.dev/dev-guide/subcomponents.html).